__tests__/test.js

```diff
@@ -35,15 +36,33 @@ describe("test DB for requests", () => {
         });
       });
   });
-  test("serves up a json representation of all the available endpoints of the api", () => {
+
+  test("serves up a json representation of all the available endpoints of the api", async () => {
+    const res = await request(app).get("/api");
+    const filePath = path.join(__dirname, "..", "endpoints.json");
+    const data = await fs.readFile(filePath, "utf-8");
+    const apiEndpoints = JSON.parse(data);
+
+    expect(res.status).toBe(200);
+    expect(res.body).toEqual(apiEndpoints);
+  });
+
+  test("/api/topics returns all topics", () => {
```

Comment:
🚨 beware, your test description has nothing to do with what the test is checking.